sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -268,157 +270,155 @@ export default class OoyalaSkinBridgeListener {
       case 'ready':
         this.skin.setState({
           playing: false,
-          loading: false
+          loading: false,
         });
         break;
       case 'playing':
         this.skin.setState({
           playing: true,
           loading: false,
-          initialPlay: this.skin.state.screenType == SCREEN_TYPES.START_SCREEN,
-          screenType: this.skin.state.contentType == CONTENT_TYPES.AUDIO ?
-            SCREEN_TYPES.AUDIO_SCREEN : SCREEN_TYPES.VIDEO_SCREEN,
-          onPlayComplete: false
+          initialPlay: this.skin.state.screenType === SCREEN_TYPES.START_SCREEN,
+          screenType: this.skin.state.contentType === CONTENT_TYPES.AUDIO
+            ? SCREEN_TYPES.AUDIO_SCREEN : SCREEN_TYPES.VIDEO_SCREEN,
+          onPlayComplete: false,
         });
         break;
       case 'loading':
         this.skin.setState({
-          loading: true
+          loading: true,
         });
         break;
       default:
         break;
     }
-  };
+  }
 
   onDesiredStateChange(e) {
-    Log.log('Desired state change received: ' + e.desiredState);
+    Log.log(`Desired state change received: ${e.desiredState}`);
     this.skin.setState({
-      desiredState: e.desiredState
+      desiredState: e.desiredState,
     });
-  };
+  }
 
   onError(e) {
     Log.log('Error received');
     this.skin.setState({
-      screenType: e.screenType == SCREEN_TYPES.AUDIO_SCREEN ?
-        SCREEN_TYPES.ERROR_SCREEN_AUDIO : SCREEN_TYPES.ERROR_SCREEN,
-      error: e
+      screenType: e.screenType === SCREEN_TYPES.AUDIO_SCREEN
+        ? SCREEN_TYPES.ERROR_SCREEN_AUDIO : SCREEN_TYPES.ERROR_SCREEN,
+      error: e,
     });
-  };
+  }
 
-  onEmbedCodeSet(e) {
+  onEmbedCodeSet() {
     Log.log('EmbedCodeSet received');
     this.skin.setState({
       screenType: SCREEN_TYPES.LOADING_SCREEN,
-      ad: null
+      ad: null,
     });
-  };
+  }
 
   onUpNextDismissed(e) {
     Log.log('SetNextVideo received');
     this.skin.setState({
-      upNextDismissed: e.upNextDismissed
+      upNextDismissed: e.upNextDismissed,
     });
-  };
+  }
 
   onSetNextVideo(e) {
     Log.log('SetNextVideo received');
     this.skin.setState({
-      nextVideo: e.nextVideo
+      nextVideo: e.nextVideo,
     });
-  };
+  }
 
   onVolumeChanged(e) {
     this.skin.setState({
-      volume: e.volume
+      volume: e.volume,
     });
-  };
+  }
 
-  onControllerKeyPressed(e) {
+  onControllerKeyPressed() {
     Log.log('Controller event received');
     this.core.handleControlsTouch();
-  };
+  }
 
   onPipToggle(e) {
-    Log.log("Received PiP Toggle: " + e.isPipActivated /*+ e.isPipButtonVisible*/);
```

Comment:
What was that? `/*+ e.isPipButtonVisible*/`

Reply:
@oSulyma could you explain please

Reply:
That's mine. I left it just as note about the fact that incoming event `onPipToggle` has this param and it can be printed into console for debug.  Of course it's not used in logic of other classes that relies on `onPipToggle` result. So it can be removed here.
Sorry for this.

Reply:
Thanks!